Sources/MapboxMaps/Style/Types/ExpressionOptions.swift

```diff
@@ -52,7 +52,7 @@ public struct FormatOptions: Codable, Equatable, ExpressionArgumentConvertible {
     public var textFont: [String]?
 
     /// Overrides the color specified by the root paint property.
-    public var textColor: StyleColor?
+    public var textColor: Value<StyleColor>?
```

Comment:
this is breaking change I supposed, I wonder if this is something we can accept @evil159 @OdNairy @pjleonard37 ?

Reply:
v11?